src/components/ResultSection/ResultSection.js

```diff
@@ -0,0 +1,110 @@
+import React,{useState} from "react";
+import "./ResultSection.css";
+import ResultsBody from "../ResultsBody/ResultsBody";
+import SolidButton from "../SolidButton/SolidButton";
+import checkboxes from "../../data/checkboxes";
+import Checkbox from "../Checkbox/Checkbox";
+import DifficultyExplanation from '../DifficultyExplanation/DifficultyExplanation';
+
+import { useNavigate } from "react-router-dom";
+const ResultSection = (props) => {
+  const navigate = useNavigate();
+
+  const [checkedItems, setCheckedItems] = useState({});
+
+  const handleChange = (event, category, predicate) => {
+    setCheckedItems({
+        ...checkedItems, [event.target.name]: {
+            checked: event.target.checked,
+            category: category,
+            predicate: predicate,
+        }
+    });
+}
+  return (
+    <div>
+      <div className="results-page-section display-desktop">
+        <ResultsBody text={props.text} checkedItems={checkedItems} />
+        <div className="filters-container">
```

Comment:
hi, so in the earlier review when i said to create a new component out of the div, I meant that since lines 28-63 and lines 69-104 are identical, we could replace each of these sections with a component like <ResultFilters> which contains the same lines as lines 28-63. sorry for not being clear earlier 😅

Reply:
I see!